noir-projects/noir-protocol-circuits/crates/rollup-lib/src/base/private_base_rollup.nr

```diff
@@ -1004,24 +1007,24 @@ mod tests {
         }
         offset += total_private_logs_len;
         // cc logs
-        let total_contract_class_logs_len = NUM_CC_LOGS * (cc_log.len() + 1);
+        let total_contract_class_logs_len = NUM_CC_LOGS * (cc_log.len() + 2);
```

Comment:
What's `+2`?

Reply:
I've added the address - this is just a test and it should be documented everywhere else. I can add it here too though?